test/unit/spec/EnrollChoices_spec.js

```diff
@@ -775,6 +828,16 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
               expect(test.form.factorCardinalityText(factorName)).toBe('(3 set up)');
             });
         });
+        itp('checks correct order after enroll', function () {
+          return setupMultipleFactorEnrollments(
+            factorName, 'ACTIVE', 'OPTIONAL',{'enrolled': 1, 'minimum': 1, 'maximum': 3}
+          ).then(function (test) {
+            var factorRows = test.form.optionalFactorList();
+            var factorIndex = factorRows.find(`[data-se="${factorName}"]`).index();
+            var expectedIndex = FactorUtil.getFactorSortOrder(factorEnrollList[factorName].provider, factorEnrollList[factorName].factorType) - 1;
```

Comment:
we do not need to pass provider for multiple enrollments. Please see above comment